### PR TITLE
fix: getItem was returning empty value string as null

### PR DESCRIPTION
**Pull request by plainweaver**

### CODE REVIEW

**Comment by gr2m** [comment]
I got it :)

File: lib/localstorage-memory.js [hunk 1]
```diff
@@ -15,7 +15,11 @@
    * @returns {String|null}
    */
   localStorageMemory.getItem = function (key) {
-    return cache[key] || null
+    if (key in cache) {
```
Comment (gr2m) [inline lib/localstorage-memory.js]: I’ve changed your implementation slightly. `if (cache[key]) { }`  still would be false if the value is `''`. But `if (key in cache) {}` works. Your test is great 👌

Comment (plainweaver) [reply lib/localstorage-memory.js]: Cool :) Is there anything left would I do?

**Comment by gr2m** [comment]
That should have happened automatically, but it failed https://travis-ci.org/gr2m/localstorage-memory/jobs/457218091 I’m looking into it

**Comment by gr2m** [comment]
^^ there we go :)

**Pull request merged**